packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -64,6 +81,15 @@ export class WorkspaceCommandContribution implements CommandContribution {
         @inject(OpenerService) protected readonly openerService: OpenerService
     ) { }
 
+
+
+
+    protected readonly onIUriChangedEmitter = new Emitter<IUriChangedEvent>();
+
+    get onIUriChanged(): Event<IUriChangedEvent> {
```

Comment:
We should move this to FileSytem and call it FileMovedEvent and it should be about what moved.

Reply:
Once the EditorManager receives it it can check if it has a editor for that URI and update it.